### PR TITLE
fix(news): improve responsive display - INNO-418

### PR DESCRIPTION
**Pull request by emeryro**
Fix responsive display

### CODE REVIEW

File: framework/components/news/_news.scss [hunk 1]
```diff
@@ -2,17 +2,19 @@
   align-items: stretch;
   background-color: #fff;
   display: flex;
+  flex-direction: column;
 
   &__image {
-    flex: 0 0 18em;
+    flex: 1;
```
Comment (yhuard) [inline framework/components/news/_news.scss]: Anti-pattern: https://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/

Please use the full rule name :)

File: framework/components/news/_news.scss [hunk 2]
```diff
@@ -22,11 +24,36 @@
...
-  flex-direction: row-reverse;
+@media (min-width: map-get($grid-breakpoints, 'md')) {
+  .news {
+    flex-direction: row;
 
-  .news__body {
-    padding-left: 0;
-    padding-right: map-get($spacing, 'm');
+    &__image {
+      flex: 0 0 12em;
```
Comment (yhuard) [inline framework/components/news/_news.scss]: I could accept it even though I'd prefer to see the 3 rules separately  👼

**Comment by emeryro** [comment]
Modification applied

File: framework/components/news/_news.scss [hunk 3]
```diff
@@ -53,7 +57,7 @@
 @media (min-width: map-get($grid-breakpoints, 'lg')) {
   .news {
     &__image {
-      flex: 0 0 18em;
+      flex-basis: 18em;
```
Comment (yhuard) [inline framework/components/news/_news.scss]: Here you lose the `flex-shrink: 1` rule, is it on purpose?

Comment (emeryro) [inline framework/components/news/_news.scss]: yep, only change between 'md' and 'lg' display is flex-basis

**Pull request merged**